recipes/gclib/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: gclib
+  version: "907a9d"
```

Comment:
I'm not sure Conda can handle this in a long run. Is the next hash smaller or greater than this one?
Can you use 0.0.0.1 or something.